rooms/models.py

```diff
@@ -0,0 +1,8 @@
+from django.core.validators import MaxValueValidator, MinValueValidator
+from django.db import models
+
+
+class Room(models.Model):
+    name = models.CharField(max_length=50)
+    contents = models.TextField()
+    maxCapacity = models.PositiveIntegerField(validators=[MinValueValidator(1), MaxValueValidator(350)])
```

Comment:
Can we change this to `capacity`? I'm a picky bitch, and technically `maxCapacity` is redundant